tools/python/find_optimizer_opset_version_updates_required.py

```diff
@@ -71,28 +71,29 @@ def get_multiline_call_args_from_file(filename, function_or_declaration):
 
     results = []
     with open(filename) as f:
-        cur_line = None
+        current_args = None
 
         for line in f:
-            if not cur_line:
+            if not current_args:
                 # look for new match
                 start = line.find(function_or_declaration)
                 if start != -1:
-                    cur_line = line
+                    # include the '(' in case that is the last character in the line
+                    current_args = line[start:].strip()
             else:
                 # append to existing line and look for closing ')'
-                start = len(cur_line)
-                cur_line += line.strip()
+                start = len(current_args)
+                current_args += line.strip()
 
             end = -1
-            if cur_line:
-                end = cur_line.find(")", start)
+            if current_args:
+                end = current_args.find(")", start)
 
-            have_all_args = cur_line and end != -1
+            have_all_args = current_args and end != -1
 
             if have_all_args:
```

Comment:
I would add `assert current_args` to make the type checker happy

Reply:
Reworked it so it doesn't complain. 

Not a fan of random statement to make a checker happy. That imposes a maintenance cost of wasted time as each developer that sees that line wonders why there's such a meaningless check, potentially removes it as it's just noise, finds the invalid pyright complaint, and changes it back.

Reply:
Thanks! This is even better